src/v1/providers/firestore.ts

```diff
@@ -96,7 +87,6 @@ export class DatabaseBuilder {
 }
 
 export class NamespaceBuilder {
```

Comment:
firestore.ts is the only thing use this. Does this need to be exported still?

Should it be @internal instread?

Reply:
I think this object ultimately gets exposed on customer code.

Marking it @internal might break `tsc` on customer code 🤔

Reply:
🤔  Leaving it then..